.github/workflows/update-readme.yml

```diff
@@ -1,11 +1,12 @@
 name: Check README parser info
 
-on: [push, pull_request]
+on: [push]
 
 jobs:
   luacheck:
     name: Check README parser info
     runs-on: ubuntu-latest
+    if: github.ref != 'master'
```

Comment:
Those two conditions seems to make this action never run...

Reply:
We should remove the `if` i guess

Reply:
It was like this before and we are not on master. What's strange is that the GH actions commit does not seem to be associated with any branch. Was it only a shallow checkup?

Reply:
> The action is run + all checks are run on the actual commit of the user. But not after the push of the GH bot.

Can you try that once just to see ?

Reply:
tried. made no difference. nvim-lspconfig has the same problem right now. Something changed with GH actions.